sdf/final-project/technical.md

```diff
@@ -6,102 +6,138 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
-- [ ] **API Documentation (if applicable)**: For projects with an API, is there clear and detailed API documentation, including endpoints, request/response formats, and authentication methods?
 
 
-### Score (/5):
+
+
+## Readme (max: 10 points)
+- [ ] **Markdown**: Is the README formatted using Markdown?
+- [ ] **Naming**: Is the repository name relevant to the project?
+- [ ] **1-liner**: Is there a 1-liner briefly describing the project?
+- [ ] **Instructions**: Are there detailed setup and installation instructions, ensuring a new developer can get the project running locally without external help?
+- [ ] **Configuration**: Are configuration instructions provided, such as environment variables or configuration files that need to be set up?
+- [ ] **Contribution**: Are there clear contribution guidelines? Do they outline how developers can contribute to the project, including coding conventions, branch naming conventions, and the pull request process?
+- [ ] **ERD**: Does the documentation include visual an ERD?
+- [ ] **Troubleshooting**: Is there an FAQs or Troubleshooting section that addresses common issues, questions, or obstacles users or new contributors might face?
+- [ ] **Screenshots**: Are there screenshots of the application in the README?
+- [ ] **API Documentation (for projects providing their own API endpoints)**: Is there clear and detailed documentation for the project's API? This should include descriptions of all endpoints, request/response formats, and authentication methods.
 
 ### Notes:
 
-## Version Control
-- [ ] **Use of Version Control System**: Is the project using a version control system such as Git?
+## Version Control (max: 10 points)
+- [ ] **Version Control**: Is the project using a version control system such as Git?
 - [ ] **Repository Management**: Is the repository hosted on a platform like GitHub, GitLab, or Bitbucket, making it accessible for collaboration and review?
-- [ ] **Branching and Merging Strategy**: Does the project employ a clear branching and merging strategy, such as Git Flow, to organize development and feature integration? (at least once)
 - [ ] **Commit Quality**: Does the project have regular commits with clear, descriptive messages?
-- [ ] **Issue Tracking and Pull Requests**: Is the project utilizing issue tracking to manage tasks and bugs, and are these issues linked to pull requests (PRs)? (at least once)
-
-### Bonus (max 2)
+- [ ] **Pull Requests**: Does the project employ a clear branching and merging strategy, such as Git Flow, to organize development and feature integration?
+- [ ] **Issues**: Is the project utilizing issue tracking to manage tasks and bugs?
+- [ ] **Linked Issues**: Are these issues linked to pull requests (PRs)? (at least once)
 - [ ] **Project Board**: Does the project utilize a project board (e.g., GitHub Projects or Trello) to manage and prioritize work items?
```

Comment:
maybe add a note here that project board must be linked to repository or readme.

Reply:
and public, otherwise we wont know it's there

Reply:
Done